### PR TITLE
Feature/95/create sponsor section and styling

### PR DESCRIPTION
**Pull request by AmirAgassi**
🤝[Create Sponsor Section and Styling #95](https://github.com/LaurierHawkHacks/Landing/issues/95)

🔍 **What's Included**
- Added new Sponsor section in `Sponsor.section.tsx`.
- Integrated Sponsor section into the landing page in `Landing.tsx`.
- Included new assets for sponsor and partner logos in the `src/assets/logos` directory.
- Updated `index.tsx` to export the new `SponsorSection`.

📁 Files Affected:
- `src/assets/logos/*`
- `src/assets/sponsor/*`
- `src/components/sections/Sponsor.section.tsx`
- `src/components/index.tsx`
- `src/pages/Landing.tsx`

### CODE REVIEW

File: src/components/sections/Sponsor.section.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+import DclLogo from '../../assets/logos/dcl.png';
+// Partner logos
+import StudentUnionLogo from '../../assets/logos/studentsunion.png';
+import CcubedLogo from '../../assets/logos/ccubed.png';
+import MlhLogo from '../../assets/logos/mlh.png';
+import FossaLogo from '../../assets/logos/fossa.png';
+import LcsLogo from '../../assets/logos/lcs.png';
+const SponsorSection: React.FC = () => {
+    return (
+        <section className="sponsor-section relative" style={{ position: 'relative', zIndex: '2' }}>
```
Comment (juancwu) [inline src/components/sections/Sponsor.section.tsx:29]:
```suggestion
        <section className="sponsor-section relative z-[2]">
```

**Review by juancwu — CHANGES_REQUESTED** [review]

**Comment by juancwu** [comment]
Was the head of the bird cut off on purpose?

**Comment by AmirAgassi** [comment]
> Was the head of the bird cut off on purpose?

Yup, both here and on the contact section, there's a section/footer above and below them that the bird's supposed to clip into in a prod environment. The bird uses absolute positioning as to not mess up any sections above/below it with a huge space.

(I just add the section twice in `Landing.tsx` to add stuff above/below, so I can make sure it actually works lol)

**Review by juancwu — APPROVED** [review]

**Comment by juancwu** [comment]
Just need to resolve the merge conflicts.

**Review by juancwu — APPROVED** [review]

**Pull request merged**